cxxfilt/__init__.py

```diff
@@ -1,117 +1,243 @@
+from __future__ import annotations
+
 import abc
 import ctypes
 import ctypes.util
-
-from typing import Sequence, Union
+from typing import Union
 
 from cxxfilt.version import __version__  # noqa
 
+__all__ = [
+    "Error",
+    "LibraryNotFoundError",
+    "InternalError",
+    "InvalidNameError",
+    "Demangler",
+    "demangle",
+    "demangleb",
+]
+
 
 class Error(Exception):
+    """Base class for all cxxfilt exceptions."""
+
     pass
 
 
-class LibraryNotFound(Error):
-    pass
+# can be subclassed from FileNotFoundError?
+# see https://stackoverflow.com/a/36077407
+class LibraryNotFoundError(Error):
```

Comment:
Also, we cannot subclass from `FileNotFoundError` (because it wasn't) as it changes observable behavior, such as

```
try:
    with open(input) as file:
        output = cxxfilt.demangle(file.read())
except FileNotFoundError:
    print(input, 'not found')
except cxxfilt.InvalidName:
    print(input, 'invalid')
else:
    print(output)
```